docs/source/quickstart.rst

```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
 Program your GoPiGo3
 ***********************
 
-For programming your `GoPiGo3`_ to do anything you want, read the instructions found here: `programming your robot`_.
+For programming your `GoPiGo3`_ to do anything you want, you can follow the instructions found here (`programming your robot`_) or *follow the rest of instructions found in this section*.
+
+For installing/updating the `GoPiGo3`_ on your RaspberryPi, you have to open up a terminal and hit the following command:
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_gopigo3 | bash
+
+Also, in order to be able to use the :py:meth:`easygopigo3.EasyGoPiGo3.init_distance_sensor` method and the :py:class:`easygopigo3.DistanceSensor` class, the `DI-Sensors`_ package is required.
+You can get it installed/updated by typing the following command:
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_sensors | bash
+
+***********************
+Connecting More Sensors
```

Comment:
Connect More Sensors
to keep the same style as
Program your GoPiGo3 just above

Reply:
Right that makes sense.